installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh

```diff
@@ -38,7 +38,7 @@ BROKER_CLIENT_KEY=$(cat /tmp/etcd-cert/MyClient1.key | base64)
 curl -s ${TEMPLATE_URL} > "${TEMPLATE_LOCAL}"
 
 oc process -f "${TEMPLATE_LOCAL}" \
--n ansible-service-broker \
+-n ansible_service_broker_namespace \
```

Comment:
This is trying to do all of this in the `ansible_service_broker_namespace` namespace. I think we want to pass in the variable as an argument and use the argument here. See https://github.com/aerogear/mobile-core/blob/b26279e1acd11d611b1fcd317aaace54c7a6fe81/installer/roles/ansible-service-broker-setup/tasks/main.yml#L43